Examples/MAX32665/SPIXF_SFCC/project.mk

```diff
@@ -0,0 +1,17 @@
+# This file can be used to set build configuration
+# variables.  These variables are defined in a file called 
+# "Makefile" that is located next to this one.
+
+# For instructions on how to use this system, see
+# https://github.com/Analog-Devices-MSDK/VSCode-Maxim/tree/develop#build-configuration
+
+#BOARD=FTHR_RevA
```

Comment:
Just noticed this - Should be `BOARD=FTHR` or `BOARD=FTHR2` for MAX32665.

You can just delete this whole comment header and replace it with the latest one:

https://github.com/Analog-Devices-MSDK/VSCode-Maxim/blob/65af7c61800c7039956f3c1971ffd7915008668d/MaximSDK/Inject/project.mk

I removed the BOARD example because it was causing too much confusion.